pkg/infradriver/ssh_infradriver.go

```diff
@@ -95,6 +96,7 @@ func ConvertEnv(envList []string) (env map[string]interface{}) {
 	return
 }
 
+// NewInfraDriver will create a new Infra driver, and if extraEnv specified, it will set env not exist in Cluster
```

Comment:
// NewInfraDriver will create a new infra driver. if extra Env was specified , it will overwrite the default value which is defined in the code.